release-schema.json

```diff
@@ -0,0 +1,107 @@
+{
+  "definitions": {
+    "Amendment": {
+      "properties": {
+        "unstructuredChanges": {
+          "title": "Unstructured changes",
+          "description": "A list of unstructured changes that apply to the procedure.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UnstructuredChange"
+          },
+          "wholeListMerge": true
+        }
+      }
+    },
+    "UnstructuredChange": {
+      "title": "Unstructured change",
+      "description": "An unstructured change.",
+      "type": [
+        "object"
+      ],
+      "properties": {
+        "relatedLot": {
+          "title": "Related lot",
+          "description": "If this change relates to a particular lot, provide the identifier of the related lot here.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "oldValue": {
+          "title": "Old value",
+          "description": "Value before the change",
+          "$ref": "#/definitions/UnstructuredChangeValue"
+        },
+        "newValue": {
+          "title": "New value",
+          "description": "Value after the change",
+          "$ref": "#/definitions/UnstructuredChangeValue"
+        },
+        "where": {
+          "title": "Where",
+          "description": "Location of the change",
+          "type": [
+            "object"
+          ],
+          "properties": {
+            "section": {
+              "title": "Section",
+              "description": "The section of the changed field",
+              "type": [
+                "string",
+                "null"
+              ]
+            },
+            "label": {
+              "title": "Label",
+              "description": "The label of the changed field",
+              "type": [
+                "string",
+                "null"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "UnstructuredChangeValue": {
+      "title": "Unstructured change value",
+      "description": "Old or new unstructured change value",
```

Comment:
> A value of a data element. The value must be described using only one of the fields `text`, `classifications` or `date`.

Reply:
Done

Reply:
I don't know if this last one was committed. Also, GitHub unhelpfully hid three comments, so you have to click "Load more..." above.

Reply:
Hmm it's strange, I really don't remember seeing so many comments, or a Load more button. Anyway, it's pushed.

Reply:
I still see "Old or new unstructured change value" instead of:

```
A value of a data element. The value must be described using only one of the fields `text`, `classifications` or `date`.
```